docs/dev-proc/standard-release-process.md

```diff
@@ -0,0 +1,53 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged to
+   into main.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Create and push tag that matches the newly bumped version.
+4. Fill up the `CHANGELOG.md` file with latest changes.
```

Comment:
Shouldn't this step be done before pushing tag?

Reply:
No, because then the changelog tool will mark the changes as `Unreleased` instead of the release version

Reply:
That's kinda weird. If you checkout tag then you'll have outdated changelog.
Are we sure we want to do release this way?

Reply:
This is based on the meta-rte release doc https://github.com/3mdeb/meta-rte/blob/master/docs/release.md#general-release-process

Reply:
Ok, i'll resolve this conversation but I think changelog should be available before tag.
Each tag is automatically added to releases, e.g. https://github.com/zarhus/meta-zarhus/tags where you can download source code. Which will contain out of date changelog.

Reply:
Let's continue this discussion. There is a workaround for this problem:
You can first create the new tag, then generate the changelog, commit it and move the existing tag to that commit.
I'm not a fan of this, but it resolved the problem.
@artur-rs cc

Reply:
I'm also not a fan of this. This workflow wouldn't work in e.g. dts where pushing tag would run release workflow. It's possible we will use similar one in Zarhus.
Here is example how to include changelog: https://github.com/CookPete/auto-changelog?tab=readme-ov-file#what-you-might-do-if-youre-clever

> Now every time you run [npm version](https://docs.npmjs.com/cli/version), the changelog will automatically update and be part of the version commit.

Maybe we can use that?

Reply:
Ok my bad, previous comments are based on old auto-changelog.
Current one: https://git-cliff.org/docs/usage/examples/

> Set a tag for the unreleased changes:

```shell
# it doesn't have to be an existing tag
git cliff --tag 1.0.0
```

```shell
# generate changelog for unreleased commits
git cliff --unreleased --tag 1.0.0
```

Reply:
The script now uses the `--tag` option. https://github.com/3mdeb/meta-rte/pull/75/commits/65aa284f45c7b824563e430aea4fc3d380d1a3a6#diff-c11bab05c2b4d598ee20cb276f0bd267a29f77a0065f59395fed6ccc2dce138fR6

Here's the updated doc:
https://github.com/zarhus/docs/compare/3340801d827e38a8c37ea7af5777b95a9b437c1a..20e8d1eb800f1e034ce33d1d4bbe2617af90d8a1